### DISCUSSION TITLE
where can i change between normal download and download via ca app ?

### DISCUSSION THREAD

**Discussion by hoernli** [aclap-dev/video-downloadhelper#562] [Help and Q&A] [answered]
### Your question

where can i change between normal download and download via ca app ? And where is the rest of the settings ? 

### Addon & CoApp version and details

Version 9.0.1.13 (stable), target mozilla, locale de.

**Comment by paulrouget** [comment, marked as answer]
In the `settings > more settings` there's an option "Do Not Use CoApp When Possible".

The rest of the settings have been removed as they have been non operational for almost a year now.

Unless I missed something, then let me know. You can always get back to the original UI if you want (in the settings: Use Legacy UI).

**Comment by GrampaWildWilly** [comment]
![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/92c9a72b-adb8-4005-9a60-0b3cff56e989)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/eb083cc6-02fc-4a02-9e38-e01dfd7e6cd1)

**Comment by GrampaWildWilly** [comment]
Oh.  You want to swap between using the CoApp & not.  My mistake.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/d8051344-cdd0-4d03-9421-92d0122a5ccb)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/b4c243fa-20e0-41e7-82d8-b77af7e87bf3)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/4e19f086-7de5-4e2d-8372-9494598f3959)
